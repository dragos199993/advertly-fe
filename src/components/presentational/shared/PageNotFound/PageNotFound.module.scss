.pageNotFound {
  margin-top: 8rem;
  font-size: 2.8rem;
  text-align: center;
}
