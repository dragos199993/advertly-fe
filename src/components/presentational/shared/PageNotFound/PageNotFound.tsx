import React, { FunctionComponent } from 'react';
import { FormattedMessage } from 'react-intl';

import styles from './PageNotFound.module.scss';

const PageNotFound: FunctionComponent = () => (
  <div className={ styles.pageNotFound }>
    <FormattedMessage id="page.notFound" />
  </div>
);

export default PageNotFound;
