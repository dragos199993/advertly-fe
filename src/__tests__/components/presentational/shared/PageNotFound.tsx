import React from 'react';
import { shallow, ShallowWrapper  } from 'enzyme';
import PageNotFound from '../../../../components/presentational/shared/PageNotFound/PageNotFound';

describe('PageNotFound', () => {
  let wrapper: ShallowWrapper<{}>;
  wrapper = shallow(<PageNotFound />);

  test('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
