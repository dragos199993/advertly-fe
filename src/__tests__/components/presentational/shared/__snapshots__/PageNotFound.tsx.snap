// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PageNotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <FormattedMessage
        id="page.notFound"
        values={Object {}}
      />,
      "className": "pageNotFound",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "id": "page.notFound",
        "values": Object {},
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <FormattedMessage
          id="page.notFound"
          values={Object {}}
        />,
        "className": "pageNotFound",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "id": "page.notFound",
          "values": Object {},
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
